addons/hr_expense/tests/test_expenses.py

```diff
@@ -1,11 +1,12 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 import base64
-from datetime import date
+from datetime import date, datetime
+
 from freezegun import freeze_time
 
-from odoo import Command
 from odoo.addons.hr_expense.tests.common import TestExpenseCommon
+from odoo import Command
```

Comment:
useless diff :p (The space between the builtin import and third party import of line 5 is ok for me as it's PEP friendly but here it's in the same (good) block and adds nothing